static/app/utils/useMutateApiToken.tsx

```diff
@@ -0,0 +1,86 @@
+import type {InternalAppApiToken} from 'sentry/types/user';
+import {
+  getApiQueryData,
+  setApiQueryData,
+  useMutation,
+  useQueryClient,
+} from 'sentry/utils/queryClient';
+import type RequestError from 'sentry/utils/requestError/requestError';
+import useApi from 'sentry/utils/useApi';
+
+const API_TOKEN_QUERY_KEY = ['/api-tokens/'] as const;
```

Comment:
shouldn't need to type cast this, but you can type it a little more:
```suggestion
const API_TOKEN_QUERY_KEY: string[] = ['/api-tokens/'];
```